packages/shared/routes/search.js

```diff
@@ -3,8 +3,12 @@ const MarkoWebSearchConfig = require('@parameter1/base-cms-marko-web-search/conf
 const middleware = require('@parameter1/base-cms-marko-web-search/middleware');
 const template = require('../templates/search');
 
-module.exports = (app, siteConfig) => {
-  const { contentTypes, assignedToWebsiteSectionIds } = getAsObject(siteConfig, 'search');
+module.exports = (app) => {
+  const { site } = app.locals;
+  const {
+    contentTypes,
+    assignedToWebsiteSectionIds,
+  } = getAsObject(getAsObject(site, 'config'), 'search');
```

Comment:
Don't nest `getAsObject` calls here, either do a single one:
```js
const { contentTypes, assignedToWebsiteSectionIds } = getAsObject(app, 'locals.site.config.search');
```
or chain it with the SiteConfig's implementation:
```js
const { contentTypes, assignedToWebsiteSectionIds } = get(app, 'locals.site').getAsObject('search');
```

However you can safely assume that `app.locals.site` will always exist, so I'd recommend this for readability:
```suggestion
  const { site } = app.locals;
  const { contentTypes, assignedToWebsiteSectionIds } = site.getAsObject('search');
```